body_defaults:
  position_noise: 0.05
  rotation_noise: 0.01
default_body: pennstock
bodies:
 - pennstock:
     is_static: true
     default_tag_size: 0.125
     pose:
       position:
         x: 0
         y: 0
         z: 0
       rotation:
         x: 0
         y: 0
         z: 0
       position_noise:
         x: 0.0001
         y: 0.0001
         z: 0.0001
       rotation_noise:
         x: 1.0e-05
         y: 1.0e-05
         z: 1.0e-05
     type: simple
     tags:
       - id: 44
         size: 0.15
         pose:
           position:
             x: 0
             y: 0
             z: 0
           rotation:
             x: 0.05732203
             y: 0.74746230
             z: 0.02373698
           position_noise:
             x: 0.001
             y: 0.001
             z: 0.001
           rotation_noise:
             x: 0.1
             y: 0.1
             z: 0.1
 - rig:
     is_static: false
     type: camera_rig
     fake_odom_translation_noise: 1.0
     fake_odom_rotation_noise: 1.0

plane_measurements:
  - plane_1:
      tags: [32, 81, 40, 24, 20, 8, 49, 95, 96, 90, 99, 47, 46, 45, 44, 86, 91, 92, 100, 83, 87, 57, 36, 55, 39, 98, 64, 97, 70, 65, 66, 67]
      distance: 0
      noise: 0.01
      direction: [0.0, 0.0, 1.0]
